backend/src/main/java/com/example/backend/rest/GatewayController.java

```diff
@@ -27,7 +27,7 @@ public GatewayController(ArticleScraperService articleScraperService, DataAnalys
         this.dataAnalysisService = dataAnalysisService;
     }
 
-    @GetMapping(value = "/call-scraping-service", consumes = {"application/json"}, produces = "application/json")
+    @PostMapping(value = "/call-scraping-service", consumes = {"application/json"}, produces = "application/json")
     public ResponseEntity<GatewayResponse> handleRequestToScrapingService(@RequestBody GatewayRequest gatewayRequest) throws IOException {
```

Comment:
wasn't changing ResponseEntity<GatewayResponse> to ResponseEntity<String> necessary to get the needed format of the response?